slices/redis-tools.yaml

```diff
@@ -0,0 +1,26 @@
+package: redis-tools
+
+essential:
+  - redis-tools_copyright
+
+slices:
+  bins:
+    # redis-tools actually depends on adduser as well to create and remove
+    # a user during package install/removal - however we don't support this
+    # currently so adduser is not added here.
+    essential:
+      - libatomic1_libs
```

Comment:
Do we need the adduser package as well?

```
ubuntu@noble:~$ apt show redis-tools
...
Depends: adduser, libatomic1 (>= 4.8), libc6 (>= 2.38), libjemalloc2 (>= 3.5.0), liblzf1 (>= 1.5), libssl3t64 (>= 3.0.0), 
...
```

Reply:
As clearly stated in the comment above:
```
    # redis-tools actually depends on adduser as well to create and remove
    # a user during package install/removal - however we don't support this
    # currently so adduser is not added here.
```

Reply:
Also, leaving out `adduser` when it is only used by `dpkg` during package install/removal has been done in other slices.
See:
- dbus-system-bus-common.yaml
- openssh-client.yaml
- openssh-server.yaml

Reply:
Thanks, sorry I did not catch that.